scopesim/effects/data_container.py

```diff
@@ -189,4 +197,121 @@ def validate(self, etype):
         etype_colname = utils.real_colname("ETYPE", self.meta.colnames)
         return self.meta[etype_colname] == etype
 
+    @property
+    def plot_filename(self):
+        plot_fname = self.meta["report"]["plot_filename"]
+        if plot_fname is None:
+            plot_fname = self.meta["name"].lower().replace(" ", "_")
+        plot_fformat = self.meta["report"]["plot_file_format"]
+
+        return plot_fname + "." + plot_fformat
+
+
+    @property
+    def table_string(self):
+        if isinstance(self.table, Table):
+            tbl_str = str(self.table).replace("-", "=")
+            hdr = tbl_str.split("\n")[1]
+            tbl_str = hdr + "\n" + tbl_str + "\n" + hdr
+        else:
+            tbl_str = ""
+
+        return tbl_str
+
+    @property
+    def meta_string(self):
+        meta_str = ""
+        max_key_len = max([len(key) for key in self.meta.keys()])
+        for key in self.meta:
+            if key not in ["comments", "changes", "description", "history", "report"]:
+                meta_str += f"    {key.rjust(max_key_len)} : {self.meta[key]}\n"
```

Comment:
I guess I'll have to stop testing in 3.5 if I want this ... Hmm. 
Will that also mean scopesim can't be imported into 3.5 by a random user?

Reply:
This indeed means that Python 3.5 users cannot import scopesim anymore. You should decide for yourself whether that's worth it.

Perhaps you should add some instructions to the readme to explain how to create virtual environments (with built-in `venv`, or `pipenv`, or `conda`.

MicadoWISE is all Python 3.8 now (because that supports symlinks in Windows), so either way is fine with me.